packages/cli-utils/src/ts/project.ts

```diff
@@ -1,5 +1,59 @@
 import type { GraphQLSPConfig } from '@gql.tada/internal';
-import type { Project } from 'ts-morph';
+import type { Project, SourceFile } from 'ts-morph';
+import * as vue from '@vue/language-core';
```

Comment:
Could be an optional peer-dep I reckon?

Reply:
This depends. The `doctor` command can in theory prompt people who use Vue to install `@vue/language-core` as a direct dependency in their project. Depends on whether that's typically already done by people commonly